nmtwizard/tokenizer.py

```diff
@@ -58,3 +63,30 @@ def tokenize(tokenizer, text):
     words,_ = tokenizer.tokenize(text)
     output = " ".join(words)
     return output
+
+def make_subword_learner(subword_config, subword_dir):
+
+    if 'params' not in subword_config:
+        raise RuntimeError('Parameter field \'params\' should be specified for subword model learning.')
+    params = subword_config['params']
+
+    if 'type' not in subword_config:
+        raise RuntimeError('\'type\' field should be specified for subword model learning.')
+    subword_type = subword_config['type']
+
+    if 'vocab_size' not in params :
+        raise RuntimeError('\'vocab_size\' should be specified for subword model learning.')
+    size = params['vocab_size']
+
+    learner = None
+    if (subword_type == "bpe"):
+        min_frequency = params['min-frequency'] if 'min-frequency' in params else 0
+        total_symbols = params['total_symbols'] if 'total_symbols' in params else False
+        # If no tokenizer is specified, the default tokenizer is space mode.
+        learner = pyonmttok.BPELearner(None, size, min_frequency, total_symbols)
```

Comment:
Either remove first `None` argument or use keyword arguments to make it clearer (same for `SentencePieceLearner`)